examples/deno.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Bot,
+  Context,
+  session,
+  SessionFlavor,
+} from "https://deno.land/x/grammy@v1.9.0/mod.ts";
+import { I18n, I18nContextFlavor } from "../src/mod.ts";
+
+interface SessionData {
+  __language_code?: string;
+  apples: number;
+}
+
+type MyContext =
+  & Context
+  & I18nContextFlavor
+  & SessionFlavor<SessionData>;
+
+const bot = new Bot<MyContext>(""); // <-- put your bot token here (https://t.me/BotFather)
+
+bot.use(session({
+  initial: (): SessionData => {
+    return { apples: 0 };
+  },
+}));
+
+const i18n = new I18n({
+  defaultLocale: "en",
+  useSession: true,
+  directory: "locales",
+});
+
+bot.use(i18n);
+
+bot.command("start", async (ctx) => {
+  await ctx.reply(ctx.t("greeting"));
+});
+
+bot.command("en", async (ctx) => {
+  // If 'useSession' is set to true, updates the session;
+  // otherwise throws an error. You can use ctx.i18n.useLocale("en")
+  // instead to suppress the error.
+  await ctx.i18n.setLocale("en");
+  await ctx.reply(ctx.t("language-set"));
+});
```

Comment:
I have an example to set multiple change-language command:

```TypeScript
bot.command(["en", "de", "ku", "ckb"], async (ctx) => {
  const requestLocale = ctx.message?.text.split(' ')[0].substring(1)

  if (!requestLocale)
    throw new Error('Missing locale code from user command!')

  await ctx.i18n.setLocale(requestLocale);
  await ctx.reply(ctx.t('language-set'))
});
```

But I don't know where is the best place to put this example... Do we have a doc page for it?

Reply:
I liked this. I think we should include it in the examples, since we have a different language changer thingy in the documentation already, and this will fit well in the examples.